src/Redirect.js

```diff
@@ -23,9 +23,9 @@ class Redirect {
   }
 
   match(path) {
-    if (this._from.test(path)) {
+    if (new RegExp(this.from).test(path)) {
```

Comment:
`new RegExp(this.from)` duplicated. consider creating variable to improve performance and avoid code duplication. even better to create the regexp during initialization so it is type checked.

Reply:
That's what I thought, too – but I ran into this error:

```
1) Redirects Config Loading (from GitHub)
       Retrieves Document from GitHub:
     TypeError: Method RegExp.prototype.exec called on incompatible receiver [object Object]
      at Proxy.exec (<anonymous>)
      at Proxy.test (<anonymous>)
      at Proxy.match (src/Redirect.js:26:20)
      at redirects.reduce (src/RedirectConfig.js:47:18)
      at Array.reduce (<anonymous>)
      at RedirectConfig.match (src/RedirectConfig.js:41:27)
      at Context.it (test/redirectconfigs.test.js:35:25)
      at process._tickCallback (internal/process/next_tick.js:68:7)
```

Reply:
I fixed it in the proxy.